manifests/charts/istio-control/istio-discovery-safe/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Can we de-Kustomize?

Using Kustomize and Helm together seems redundant.

Reply:
+1 

However IMO we should first make an exact copy of the original chart, and start removing/changing in separate PRs, so there is a clear history of what is different and we can review changes. 

For initial PR we just need to agree on the name - I don't like 'istio-discovery-safe' - we can just add manicests/charts/istiod-safe ( and rename later ).

The istio-control/istio-discovery is an artifact from the ancient days when we had many separate charts before istiod.

Reply:
+1 to Costin; I'm all for removing kustomize; but we should prioritize a clear diff history where we can.